### PR TITLE
login_fb_api

### PR DESCRIPTION
**Pull request by nhattien2307**
https://edu-redmine.sun-asterisk.vn/issues/9140

### CODE REVIEW

**Comment by nhattien2307** [comment]
ready

File: app/controllers/users/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  # The path used after resending confirmation instructions.
+  # def after_resending_confirmation_instructions_path_for(resource_name)
+  #   super(resource_name)
+  # end
+
+  # The path used after confirmation.
+  # def after_confirmation_path_for(resource_name, resource)
+  #   super(resource_name, resource)
+  # end
```
Comment (VuThiTranVan) [inline app/controllers/users/confirmations_controller.rb]: xóa code comment đi chú

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```
Comment (VuThiTranVan) [inline config/routes.rb]: ` :controllers => ` -> `controllers:`

` :omniauth_callbacks => ` -> ` omniauth_callbacks:`

**Comment by nhattien2307** [comment]
ready

**Pull request merged**